src/core/perplexity_client.py

```
@@ -0,0 +1,154 @@
+        """
+        try:
+            response = requests.post(
+                f"{self.BASE_URL}/chat/completions",
+                headers=self.headers,
+                json={
+                    "model": "sonar",
+                    "messages": [{"role": "user", "content": "Hi"}],
+                },
+                timeout=15,
+            )
+            return response.status_code == 200
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)